chevah_build

```diff
@@ -15,6 +15,8 @@
 UBUNTU_PACKAGES="gcc libssl-dev zlib1g-dev libbz2-dev m4"
 RHEL_PACKAGES="gcc openssl-devel zlib-devel bzip2-devel m4"
 SLES_PACKAGES="gcc openssl-devel zlib-devel libbz2-devel m4"
+# Currently, we use Debian 7 as a generic Linux distro.
+LINUX_PACKAGES="gcc libssl-dev zlib1g-dev libbz2-dev m4"
```

Comment:
this looks similar to Ubuntu ... the LINUX_PACKAGES are in fact debian package which are similar to Ubuntu..

We are doing all work on Debian, just that we can the final product linux...